Test/testTitleData.json

```diff
@@ -0,0 +1,7 @@
+{
+    "titleId": "",
+    "secretKey": "",
+    "connectionString": "",
+    "steamAppId": "",
+    "steamPublisherKey": ""
+}
```

Comment:
I believe this will get deleted by the release pipeline automatically.  For Switch, I added this to the dev repo under \pipelines\releaseFiles and copied it to the correct spot as part of the pipeline